experiments/run_experiment.py

```diff
@@ -159,51 +164,64 @@ def validate_args(args: argparse.Namespace, parser: argparse.ArgumentParser):
         )
 
 
-def _get_dataset(
+def run_mrmc(
+    run_seed: int,
+    step_size: float,
+    confidence_cutoff: Optional[float],
+    noise_ratio: Optional[float],
+    rescale_ratio: Optional[float],
+    volcano_degree: float,
+    volcano_cutoff: float,
+    num_clusters: int,
+    max_iterations: int,
     dataset_name: str,
-) -> Tuple[pd.DataFrame, base_loader.DatasetInfo]:
-    """Gets the dataset. Useful for testing."""
-    return data_loader.load_data(data_loader.DatasetName(dataset_name))
+    model_type: str,
+    cluster_seed: int,
+    **_unused_kwargs: Any,
+) -> Tuple[Sequence[pd.DataFrame], pd.DataFrame]:
+    """Runs MRMC using the given configurations.
 
+    Args:
+        run_seed: The seed used in the run. All random numbers are derived from
+            this seed except the clustering, which uses cluster_seed.
+        step_size: The step size to use when rescaling the recourse.
+        confidence_cutoff: The target model confidence.
+        noise_ratio: The optional ratio of noise to add.
+        rescale_ratio: The optional ratio by which to rescale the direction.
+        volcano_degree: The degree to use in the MRM volcano alpha function.
+        volcano_cutoff: The cutoff to use in the MRM volcano alpha function.
+        num_clusters: The number of clusters to use when generating paths.
+        max_iterations: The maximum number of iterations to take recourse steps
+            for.
+        dataset_name: The name of the dataset to use.
+        model_type: The type of model to use.
+        cluster_seed: The seed to use for clustering.
+        _unused_kwargs: An argument used to capture kwargs from run_config that
+            aren't used by this function.
 
-def _get_recourse_adapter(
-    dataset: pd.DataFrame,
-    dataset_info: base_loader.DatasetInfo,
-    random_seed: Optional[int],
-    noise_ratio: Optional[float],
-    rescale_ratio: Optional[float],
-) -> recourse_adapter.RecourseAdapter:
-    """Gets the recourse adapter. Useful for testing."""
-    return continuous_adapter.StandardizingAdapter(
+    Returns:
+        A list of recourse paths and a dataframe containing cluster info."""
+    # generate random seeds
```

Comment:
Nit: Capitalize first letter and use full-stop punctuation for inline and other comments.

Reply:
I fixed it in this file! And will be more careful with it in future files.

I did not fix it repo-wide.